models.yml

```diff
@@ -3734,6 +3731,49 @@ mediafile:
     to: mediafile/parent_id
     equal_fields: owner_id
     restriction_mode: A
+  owner_id:
+    type: generic-relation
+    to:
+      - meeting/mediafile_ids
+      - organization/mediafile_ids
+    restriction_mode: A
+    required: true
+    constant: true
+  meeting_mediafile_ids:
+    type: relation-list
+    to: meeting_mediafile/mediafile_id
+    on_delete: CASCADE
+    restriction_mode: A
+
+meeting_mediafile:
+  id:
+    type: number
+    restriction_mode: A
+    constant: true
+  mediafile_id:
+    type: relation
+    to: mediafile/meeting_mediafile_ids
+    required: true
+    restriction_mode: A
+  meeting_id:
+    type: relation
+    to: meeting/meeting_mediafile_ids
+    required: true
+    restriction_mode: A
+  is_public:
+    type: boolean
+    description: "Calculated in actions. inherited_access_group_ids == [] can have two causes: cancelling access groups (=> is_public := false) or no access groups at all (=> is_public := true)"
+    required: true
+    restriction_mode: A
+  inherited_access_group_ids:
+    type: relation-list
+    to: group/mediafile_inherited_access_group_ids
```

Comment:
What do you think of explaining the calculation rules here?

Reply:
Good idea, will do just that.